go/kbfs/libkbfs/init.go

```diff
@@ -660,11 +660,6 @@ func doInit(
 			return lg
 		}, params.StorageRoot, params.DiskCacheMode, kbCtx)
 	config.SetVLogLevel(kbCtx.GetVDebugSetting())
-	if mode == InitConstrained {
```

Comment:
we technically don't yet have this - do you want to wait to enable this until after the GUI version of this setting is in?

Reply:
Meh I say we just go for it.  If that doesn't get in, we can revisit right before the release.